website/snippets/_new-sl-setup.md

```diff
@@ -14,12 +14,7 @@ Now that we've created and successfully run a job in your environment, you're re
 
 <Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
```

Comment:
this is a prerequ but is it still necessary?

- Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions. 
  - dbt Core or Developer accounts can define metrics but won't be able to dynamically query them.<br />

Reply:
ok to delete